### PR TITLE
NEST-113: Updating Configuration.md with info about changing HTML validation rules for specific tests

### PR DESCRIPTION
**Pull request by DemeSzabolcs**
[NEST-113](https://lombiq.atlassian.net/browse/NEST-113)

## Summary by CodeRabbit

- **Documentation**
	- Updated the Configuration guide with instructions for HTML validation setup.
	- Included guidance on multi-process test execution for improved testing practices.

[NEST-113]: https://lombiq.atlassian.net/browse/NEST-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Piedone** [comment]
@coderabbitai review

I'm testing the AI code review tool [CodeRabbit](https://coderabbit.ai/) here. Please bear with me. And if it comments some useful feedback, please address it.

**Comment by Piedone** [comment]
@coderabbitai review

Sorry about that.

File: Lombiq.Tests.UI/Docs/Configuration.md [hunk 1]
```diff
@@ -57,6 +57,28 @@ Recommendations and notes for such configuration:
 - If you have several UI test projects it can be cumbersome to maintain a _TestConfiguration.json_ file for each. Instead you can set the value of the `LOMBIQ_UI_TESTING_TOOLBOX_SHARED_TEST_CONFIGURATION` environment variable to the absolute path of a central configuration file and then each project will look it up. If you place an individual _TestConfiguration.json_ into a test directory it will still take precedence in case you need special configuration for just that one.
 - `MaxParallelTests` sets how many UI tests should run at the same time. It is an important property if you want to run your UI tests in parallel, check out the inline documentation in [`OrchardCoreUITestExecutorConfiguration`](../Services/OrchardCoreUITestExecutorConfiguration.cs).
 
+### HTML validation configuration
+
+If you want to change some HTML validation rules from only a few specific tests, you can create a custom `.htmlvalidate.json` (e.g. `TestName.htmlvalidate.json`), then you can change the configuration to use that:
```
Comment (wAsnk) [inline Lombiq.Tests.UI/Docs/Configuration.md]:
```suggestion
If you want to change some HTML validation rules from only a few specific tests, you can create a custom `.htmlvalidate.json` file (e.g. `TestName.htmlvalidate.json`), then you can change the configuration to use that:
```

Comment (wAsnk) [inline Lombiq.Tests.UI/Docs/Configuration.md]: Could you include an example for this json file?

File: Lombiq.Tests.UI/Docs/Configuration.md [hunk 2]
```diff
@@ -57,6 +57,28 @@ Recommendations and notes for such configuration:
...
+
+```cs
+ changeConfiguration: configuration => configuration.HtmlValidationConfiguration.HtmlValidationOptions =
+                configuration.HtmlValidationConfiguration.HtmlValidationOptions
+                    .CloneWith(validationOptions => validationOptions.ConfigPath =
+                        Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "TestName.htmlvalidate.json")));
+```
+
+Make sure to also include the `root` attribute and set it to `true` inside the custom `.htmlvalidate.json` and include it in the test project like this:
```
Comment (wAsnk) [inline Lombiq.Tests.UI/Docs/Configuration.md]:
```suggestion
Make sure to also include the `root` attribute and set it to `true` inside the custom `.htmlvalidate.json` file and include it in the test project like this:
```

**Review by wAsnk — CHANGES_REQUESTED** [review]

**Review by wAsnk — APPROVED** [review]

**Pull request merged**